### DISCUSSION TITLE
Labs support added

### DISCUSSION THREAD

**Discussion by fxstein** [gopro/labs#91] [Show and tell]
Hi there,

Finally got around to adding support for labs firmware to my little MacOS GoPro manager.

https://github.com/fxstein/GoProX

You can now switch to the latest labs firmware via

```goprox --firmware labs```

and back via

```goprox --firmware```

It can be part of a full workflow eg:

```goprox --archive --import --clean --time --firmware labs```

Thanks for the awesome features labs enables!

**Comment by dnewman-gpsw** [comment]
Thanks for support it.

**Reply by fxstein** [reply]
You are welcome! Lots of progress made on the tool in recent weeks. Just working through 5000 miles of footage we shot on 10 GoPros and several iPhones.
